x/ccv/provider/keeper/keeper.go

```diff
@@ -990,3 +990,25 @@ func (k Keeper) GetFirstVscSendTimestamp(ctx sdk.Context, chainID string) (vscSe
 
 	return types.VscSendTimestamp{}, false
 }
+
```

Comment:
A quick way to reduce state bloat would be to delete the slash log once the prop is executed

Reply:
There's 175 validators. State bloat is not an issue